### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by ShaliniNayan**
This PR is raised to implement the following tasks:

- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
     - Create and run the necessary migration files.
     - Table and column names should match the ERD diagram.
            - Note: photo for users table should be a link to an image
     - Foreign keys should be included.
           - Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
     - All columns that are foreign keys should have a corresponding index.
- No linters error.
- Gitflow follows.

### CODE REVIEW

**Comment by nahnahdev** [comment]
## Project Status: INVALIDATED :red_circle: 
Hi @ShaliniNayan , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that is missing the [Microverse README Template](https://github.com/microverseinc/readme-template). This makes your project incomplete as every section of the readme template is required except for the FAQ section which is optional. So please make sure to add all the Microverse README template to your project and fill in all the necessary contents :+1:

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+### Prerequisites
+
+In order to run this project you need:
+
+Example command:
+
+```sh
+ Install Ruby
+ Install rails
+```
```
Comment (Gambit142) [inline README.md]: - [OPTIONAL] Great work creating the README as directed by the previous reviewer. For this command, it is not `Install Ruby` or `Install rails` but `bundle install`. I would advise you to correct this as proper instructions help people who clone your repository know how to run it locally. 👍 😎

**Review by Gambit142 — APPROVED** [review]
Hi @ShaliniNayan 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters correctly ✔️ 
- Created migration files for the `User` model ✔️ 
- Created migration files for the `Post` model ✔️ 
- Created migration files for the `Comment` model ✔️ 
- Created migration files for the `Like` model ✔️ 

### Optional Suggestion

#### Kindly consider the comment tagged _**[OPTIONAL]**_ under this review summary. 🖱️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ShaliniNayan) [reply README.md]: Thank you & noted

**Pull request merged**